content/en/docs/contribute/_index.md

```diff
@@ -13,68 +13,40 @@ we're happy to have your help! Anyone can contribute, whether you're new to the
+of roles and permissions.
 
-This list is divided into things anyone can do, things Kubernetes organization
-members can do, and things that require a higher level of access and familiarity
-with SIG Docs processes. Contributing consistently over time can help you
-understand some of the tooling and organizational decisions that have already
-been made.
+Kubernetes documentation is managed using a GitHub repository. While we welcome
+contributions from anyone, you do need basic comfort with git and GitHub to
+operate well in the Kubernetes community.
 
-This is not an exhaustive list of ways you can contribute to the Kubernetes
-documentation, but it should help you get started.
+At a minimum, do the following to get involved with documentation:
 
-- [Anyone](/docs/contribute/start/)
-  - Open actionable issues
-- [Member](/docs/contribute/start/)
-  - Improve existing docs
-  - Bring up ideas for improvement on [Slack](http://slack.k8s.io/) or the [SIG docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
-  - Improve docs accessibility
-  - Provide non-binding feedback on PRs
-  - Write a blog post or case study
-- [Reviewer](/docs/contribute/intermediate/)
-  - Document new features
-  - Triage and categorize issues
-  - Review PRs
-  - Create diagrams, graphics assets, and embeddable screencasts / videos
-  - Localization
-  - Contribute to other repos as a docs representative
-  - Edit user-facing strings in code
-  - Improve code comments, Godoc
-- [Approver](/docs/contribute/advanced/)
-  - Publish contributor content by approving and merging PRs
-  - Participate in a Kubernetes release team as a docs representative
-  - Propose improvements to the style guide
-  - Propose improvements to docs tests
-  - Propose improvements to the Kubernetes website or other tooling
+1. Sign the CNCF [Contributor License Agreement](https://github.com/kubernetes/community/blob/master/CLA.md).
+2. Familiarize yourself with the [documentation repository](https://github.com/kubernetes/website) and the static site generator
+we use for the documentation site, [Hugo](https://gohugo.io/).
+3. Understand the basic processes around [improving content](https://kubernetes.io/docs/contribute/start/#improve-existing-content) and [reviewing changes](https://kubernetes.io/docs/contribute/start/#review-docs-pull-requests).
 
-
-## Additional ways to contribute
+## Other ways to contribute
 
 - To contribute to the Kubernetes community through online forums like Twitter or Stack Overflow, or learn about local meetups and Kubernetes events, visit the [Kubernetes community site](/community/).
 - To contribute to feature development, read the [contributor cheatsheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) to get started.
 
 {{% /capture %}}
+
+{{% capture whatsnext %}}
+
+- To read more about the basics of contributing to documentation, read [Start contributing](/docs/contribute/start/).
```

Comment:
Eliminate "read" twice in one sentence and connect the action to the purpose more explicitly.

```suggestion
- For more information about the basics of contributing to documentation, read [Start contributing](/docs/contribute/start/).
```